CHANGELOG

```diff
@@ -1,6 +1,8 @@
 devel
 -----
 
+* Changed arangosh directory name to client-tools
```

Comment:
```suggestion
* Changed "arangosh" directory name to "client-tools", because the directory contains
  the code for all client tools and not just arangosh.
```